src/DynamoUtilities/AssemblyHelper.cs

```diff
@@ -34,20 +34,29 @@ public Assembly ResolveAssembly(object sender, ResolveEventArgs args)
             try
             {
                 var targetAssemblyName = new AssemblyName(args.Name).Name + ".dll";
-
+                Console.WriteLine("trying to resolve");
+                Console.WriteLine(targetAssemblyName);
                 // First check the core path
                 string assemblyPath = Path.Combine(moduleRootFolder, targetAssemblyName);
+                Console.WriteLine("at");
+                Console.WriteLine(assemblyPath);
```

Comment:
can the console.write be in testmode ?

Reply:
we could but assembly helper class does not have a test mode(I dont think) - guess we could add one.